robot/web/web.go

```diff
@@ -322,10 +322,10 @@ func (svc *webService) StartModule(ctx context.Context) error {
 	if err := svc.modServer.RegisterServiceServer(ctx, &pb.RobotService_ServiceDesc, grpcserver.New(svc.r)); err != nil {
 		return err
 	}
-	if err := svc.refreshResources(); err != nil {
+	if err := svc.initAPIResourceCollections(ctx, true); err != nil {
```

Comment:
I don't see why refresh should happen after init, and this allows some simplification in updateResources